### PR TITLE
ignore, don't reject block if we can't determine power

### PR DESCRIPTION
**Pull request by vyzo**
Our SQA team (aka the onions) has complained about that code, and upon inspection it seems that we might erroneously reject a block message when we can't determine power.
This remedies the situation by signalling a soft failure in that case.

### CODE REVIEW

File: chain/sub/incoming.go [hunk 1]
```diff
@@ -473,19 +478,19 @@ func (bv *BlockValidator) checkPowerAndGetWorkerKey(ctx context.Context, bh *typ
 	baseTs := bv.chain.GetHeaviestTipSet()
 	lbts, err := stmgr.GetLookbackTipSetForRound(ctx, bv.stmgr, baseTs, bh.Height)
 	if err != nil {
-		log.Warnf("failed to load lookback tipset for incoming block")
-		return address.Undef, err
+		log.Warnf("failed to load lookback tipset for incoming block: %s", err)
+		return address.Undef, ErrSoftFailure
```
Comment (Kubuxu) [inline chain/sub/incoming.go:482]: Don't swallow errors into the log.
Use `xerrors.Errorf("something something: %w")` and `errors.Is(err, ErrSoftFailure)` above.

Comment (vyzo) [reply chain/sub/incoming.go:482]: sure, but we don't do anything with the error!

**Review by Kubuxu — APPROVED** [review]

**Comment by vyzo** [comment]
@schomatis yes, but we may fail to load the lookback tipset to determine power if we are not fully sycned, hence the soft failure in these two paths. The rest of the checks still happens, so if we are nearly synced we must know the actor (because it can't possibly have power with the sealing delay).

**Comment by vyzo** [comment]
Yeah maybe we should, I can't think of a good name right now however :)

**Review by magik6k — APPROVED** [review]

**Pull request merged**